readme.md

```diff
@@ -743,3 +743,19 @@ What's the best tool for automatically ignoring long email threads about tech bu
 **A:** No idea. They just ransomware.
 
 ---
+
+
+**Q:** Why did the data engineer get fired from his job as a hat maker? 
+
+**A:** He could only make two thirds of a cap!
```

Comment:
I don't get it 🤷‍♂

Reply:
Maybe this could help? https://en.wikipedia.org/wiki/CAP_theorem